File: mobile/src/main/java/com/hotpodata/redchain/adapter/viewholder/RowChainViewHolder.java
package com.hotpodata.redchain.adapter.viewholder;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import com.hotpodata.redchain.view.CircleImageView;

/**
 * Created by jdrotos on 1/21/15.
 */
public class RowChainViewHolder extends RecyclerView.ViewHolder {
    public TextView mTextOne;
    public TextView mTextTwo;
    public CircleImageView mIcon;

    public RowChainViewHolder(View itemView) {
        super(itemView);
        mTextOne = (TextView) itemView.findViewById(android.R.id.text1);
        mTextTwo = (TextView) itemView.findViewById(android.R.id.text2);
        mIcon = (CircleImageView) itemView.findViewById(android.R.id.icon);
    }
}

File: mobile/src/main/java/com/hotpodata/redchain/adapter/viewholder/RowChainViewHolder.kt
package com.hotpodata.redchain.adapter.viewholder

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.TextView

import com.hotpodata.redchain.view.CircleImageView

/**
 * Created by jdrotos on 1/21/15.
 */
class RowChainViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    var mTextOne: TextView
    var mTextTwo: TextView
    var mIcon: CircleImageView

    init {
        mTextOne = itemView.findViewById(android.R.id.text1) as TextView
        mTextTwo = itemView.findViewById(android.R.id.text2) as TextView
        mIcon = itemView.findViewById(android.R.id.icon) as CircleImageView
    }
}
